modules/components/analytics/web/org.wso2.carbon.pc.analytics.web/src/web/js/actions.js

```diff
@@ -201,7 +248,7 @@ function drawProcessInstanceCountVsProcessVersionResult(renderElement){
                         });
                     }
                     render(renderElementID, dataset, 'Process Instance Count', 'Process Version');
-                }else{
+                } else {
                     //Error message
```

Comment:
If there's nothing to add in the else scope, better to remove the else section. No point having an empty scope.